operatorcourier/identify.py

```diff
@@ -1,16 +1,27 @@
-import yaml
+from yaml import safe_load
+from yaml import MarkedYAMLError
+import logging
+
+logger = logging.getLogger(__name__)
+
 
 def get_operator_artifact_type(operatorArtifactString):
-    """get_operator_artifact_type takes a yaml string and determines if it is one of the 
+    """get_operator_artifact_type takes a yaml string and determines if it is one of the
     expected bundle types: ClusterServiceVersion, CustomResourceDefinition, or Package.
 
     :param operatorArtifactString: Yaml string to type check
     """
-    operatorArtifact = yaml.safe_load(operatorArtifactString)
-    if isinstance(operatorArtifact, dict):
-        if "packageName" in operatorArtifact:
-            return "Package"
-        elif "kind" in operatorArtifact:
-            if operatorArtifact["kind"] in ("ClusterServiceVersion", "CustomResourceDefinition"):
-                return operatorArtifact["kind"]
-    raise ValueError('Courier requires valid CSV, CRD, and Package files')
+    try:
+        operatorArtifact = safe_load(operatorArtifactString)
+    except MarkedYAMLError:
+        logger.error("Invalid YAML file")
```

Comment:
Can we mention which file couldn't be parsed here?

Reply:
This may require changes in multiple function signatures. I instead log all YAMLs that are parsed successfully.